cpp/models/abm/README.md

```diff
@@ -1,57 +1,127 @@
 
-The result of the simulation is for each time step the count of persons in each infection state at that time.
+Another way of migration we use in the simulation of Braunschweig (simulations/abm_braunschweig.cpp) is using trips. A trip consists of the ID of the person that performs this trip, a timepoint when this trip is performed and where the person is heading to. In the beginning of the simulation a list with all trips is initialized and followed during the simulation. There can be different trips on the weekend than during the week but other than that the agents do the same trips every day assuming they are not in quarantine or in the hospital.
+
+## Get Started
+
+This section gives an introduction how to use the ABM and setup your own simulation. For a quick overview can find a full example in the [abm minimal example](../../examples/abm_minimal.cpp) and a more detailed doxygen documentation [here](https://dlr-sc.github.io/memilio/documentation/index.html). For a guide on installation and running the simulations and examples see this [README](../../README.md).
 
-## Example
-An example can also be found at simulations/abm.cpp
+Every person in the ABM belongs to an AgeGroup which we can define:
+
+```cpp
+size_t NUM_AGE_GROUPS         = 4;
+const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(NUM_AGE_GROUPS - 4);
 ```
-    size_t NUM_AGE_GROUPS         = 4;
-    const auto AGE_GROUP_0_TO_4   = mio::AgeGroup(NUM_AGE_GROUPS - 4);
-    const auto AGE_GROUP_5_TO_14  = mio::AgeGroup(NUM_AGE_GROUPS - 3);
-    const auto AGE_GROUP_15_TO_34 = mio::AgeGroup(NUM_AGE_GROUPS - 2);
-    const auto AGE_GROUP_35_TO_59 = mio::AgeGroup(NUM_AGE_GROUPS - 1);
 
-    // Create the world with 4 age groups.
-    auto world = mio::abm::World(NUM_AGE_GROUPS);
-    //setup the parameters
-    world.parameters.get<mio::abm::IncubationPeriod>() = 4.;
-    //...
+The initial empty world is created with the number of age groups:
+
+```cpp
+auto world = mio::abm::World(NUM_AGE_GROUPS);
+```
 
-    //create the world with some nodes and persons
-    mio::abm::Location& home = world.add_location(mio::abm::LocationType::Home);
-    mio::abm::Location& school = world.add_location(mio::abm::LocationType::School);
-    mio::abm::Location& work = world.add_location(mio::abm::LocationType::Work);
-    mio::abm::Person& p1 = world.add_person(home, mio::abm::InfectionState::Susceptible);
-    mio::abm::Person& p2 = world.add_person(home, mio::abm::InfectionState::Susceptible);
-    mio::abm::Person& p3 = world.add_person(home, mio::abm::InfectionState::InfectedNoSymptoms);
-    mio::abm::Person& p4 = world.add_person(home, mio::abm::InfectionState::Susceptible);
+We can set a lot of general parameters. Here is an example where we set the duration of the incubation period to 4 days:
```

Comment:
Would it make sense to link to the parameters.h file, as a list of all parameters? Or maybe refer to the doxygen documentation.